spel/scripts/amigen7-build.sh

```diff
@@ -205,6 +207,18 @@ bash -eux -o pipefail "${ELBUILD}"/MkChrootTree.sh "${DEVNODE}" "" "${AMIGENSTOR
 echo "Executing MkTabs.sh"
 bash -eux -o pipefail "${ELBUILD}"/MkTabs.sh "${DEVNODE}"
 
+# Construct the cli option string for alternate-manifest
+CLIOPT_ALTMANIFEST=""
+if [[ -n ${AMIGENMANFST} ]]
```

Comment:
I'm surprised shellcheck didn't flag this...
```suggestion
if [[ -n "${AMIGENMANFST}" ]]
```

Reply:
Doublequotes is mostly for `[ ]` test blocks. Doublequotes inside `[[ ]]` testblocks is only when you specifically need them for the RHS of the test (e.g. most frequently where the RHS is `""` or something with spaces in it).